po/de.po

```diff
@@ -13070,7 +13038,7 @@ msgstr "benutzerdefinierte Sortierung"
 #. DT_COLLECTION_SORT_ASPECT_RATIO
 #: ../src/libs/tools/filter.c:158
 msgid "shuffle"
-msgstr ""
+msgstr "mischen"
```

Comment:
Ist "mischen" hier eigentlich richtig? Nicht eher sortieren?

Reply:
Ja, wahrscheinlich ist "mischen" hier falsch. Wenn ich das richtig sehe, wird dieser String im Culling-Modus verwendet. Vermute "manuelle Sortierung" oder so ähnlich wäre passend. Müsste mich, da zu beurteilen, aber erst einmal genauer mit dem Culling-Modus befassen.

Reply:
manuelle Sortierung oder "manuelle Ordnung" sind denke ich richtig.

Reply:
Habe die Übersetzung in _manuelle Sortierung_ geändert.